articles/connecting-library-application-to-sql-server-using-windows-forms/index.md

```diff
@@ -0,0 +1,232 @@
+# Connecting-library-application-to-SQL-Server-using-Windows-forms
+
+### Introduction
+
+windows forms is one of the best tools used in making applications.
+With windows forms, you can make a variety of applications, from a mini calculator to a library application that connects to a database. In this tutorial, we will connect a library application using Windows forms to a database using SQL Server.
+
+In this tutorial, we will use a windows forms project and work on it, we will add new forms that will add edit & delete books from the library, we will also manage the authors, the stores & the publishers stored in the database. We will display the information of each attribute & refresh it each time a change accrues.

```

Comment:
```suggestion
Windows forms are one of the best tools used in making applications. With windows forms, you can make a variety of applications, from a mini calculator to a library application that uses a database.

In this tutorial, we will connect a library application to a SQL database server using Windows forms. We will use a windows forms project to add new forms, edit or delete from the library. Also, we will manage the authors, the stores & the publishers in the database, and display the information of each attribute & refresh it each time, a change accrues.

```

Reply:
Please explain what a `Windows form` is. Add them in the first para. Give a 1 -2 line intro.

If possible, add a link.